build/bump.php

```diff
@@ -248,12 +251,14 @@ function usage($command)
 }
 
 // Updates the version in the package.json file.
-if (file_exists($rootPath . $packageJsonFile)) {
-    $package          = json_decode(file_get_contents($rootPath . $packageJsonFile));
-    $package->version = $version['release'];
+foreach ($packageJsonFiles as $packageJsonFile) {
+    if (file_exists($rootPath . $packageJsonFile)) {
+        $package          = json_decode(file_get_contents($rootPath . $packageJsonFile));
+        $package->version = $version['release'];
 
-    // @todo use a native formatter whenever https://github.com/php/php-src/issues/8864 is resolved
-    file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)));
+        // @todo use a native formatter whenever https://github.com/php/php-src/issues/8864 is resolved
+        file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)));
```

Comment:
```suggestion
        file_put_contents($rootPath . $packageJsonFile, str_replace('    ', '  ', json_encode($package, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)) . '/n');
```

The Code Style for JS/JSON requires an empty line at the end of the file

Reply:
That’s right.